### ISSUE TITLE
Alert/Alarm cannot be trigger when internet lose 

### ISSUE THREAD

**Issue by melancholyhua** [alexa/avs-device-sdk#913]
Hi, 

Using AVS SDK-v1.8.1. with ARM based OpenWRT embedded Linux.

Following this case, 
 
1. Set an alarm (multi-turn).
2. After being asked "for what time," answer the time that is a few minutes from now.
3. Once alarm is set, disconnect the device from the internet. 
4. Confirm internet is not connected by "Alexa what is the time" where the device responds with a "no Internet connection" error message.

At the scheduled time, the alarm sounds not trigger, because device lost the internet which is mean AVS can't receive any notification from Amazon server ? So is this the expected behavior of the SDKv1.8.1 ? Thanks.

By the way if I kill the SampleApp directly and then launched  it again, the alert will be triggered.

**Comment by visayamv** [comment]
Hi @melancholyhua , it is expected that the default alarm sound (stored locally) will be played if it fails to play the online audio sound. I have filed an issue internally to our team to further investigate and provide resolution.

**Comment by melancholyhua** [comment]
Hi @visayamv , thanks for your clarification.
I think the problem is why the alert was not triggered at the scheduled time.  As you mentioned, the expected that should be played local sound when internet lose case. but I can't see any log at the scheduled time... thanks.

**Comment by celinval** [comment]
Hi @melancholyhua, are you still facing the same issue?

**Comment by melancholyhua** [comment]
Hi @celinval ,
I didn't update to SDKv1.9 yet, so I am do workaround for this issue on SDKv1.8.1

**Comment by scotthea-amazon** [comment]
Hi @melancholyhua,

I just tried to reproduce this problem with the 1.10 release, and I thought I was seeing a similar problem.  However, when I waited about 55 seconds after the appointed time, the default alert started playing.

Looking at the log I see that when the music alarm was set (e.g. "Alexa, wake me at 6am with the Beatles on iHeart radio."), a SetAlert directive was received with a time of 6:00:55.  Upon further experimentation, it seems that the +55 second time is used to specify a fallback alarm. If the AVS connection is healthy at the specified time of the music alarm, AudioPlayer.Play directives are received to play the music, and once the SDK reports to Alexa service that the audio is playing, the alert scheduled at t +55 second time is deleted.   If the connection to AVS is not healthy, the previously specified alarm is not deleted and plays at the specified time +55 seconds.

I wonder if in your testing you waited long enough for the fallback alarm to be played.

Best regards,
-SWH
